lxd/container_console.go

```diff
@@ -275,7 +275,7 @@ func containerConsolePost(d *Daemon, r *http.Request) Response {
 	}
 
 	if client != nil {
-		url := fmt.Sprintf("/containers/%s/console", name)
+		url := fmt.Sprintf("/containers/%s/console?project=%s", name, project)
```

Comment:
Can this be made unconditional even when talking to older daemons that are not project aware?

Reply:
It can't be talking to an older daemon, this code is cluster-only and cluster members must all run the same version